logservice/logpuller/region_failure_handler.go

```
@@ -43,25 +45,159 @@ var (
 type regionFailureHandler struct {
 	cache       *errCache
 	regionCache *tikv.RegionCache
+	recovery     struct {
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gofmt)